app/screens/send-bitcoin-screen/payment-destination/intraledger.ts

```diff
@@ -24,8 +24,17 @@ export const resolveIntraledgerDestination = async ({
   accountDefaultWalletQuery,
   myWalletIds,
 }: ResolveIntraledgerDestinationParams): Promise<ParseDestinationResult> => {
-  const { handle } = parsedIntraledgerDestination
+  const { valid } = parsedIntraledgerDestination
 
+  if (!valid) {
+    return {
+      valid: false,
+      invalidReason: InvalidDestinationReason.WrongDomain,
+      invalidPaymentDestination: parsedIntraledgerDestination,
+    }
+  }
+
+  const handle = parsedIntraledgerDestination.handle
```

Comment:
better to be consistent in how we parse an object, specially in the same file. ie: either use destructuring { valid } or use .handle.